parser_test.go

```diff
@@ -1789,13 +1789,13 @@ func (s *testParserSuite) TestDDL(c *C) {
 		{"alter table t analyze partition a index b with 4 buckets", true, ""},
 
 		// For create index statement
-		{"CREATE INDEX idx ON t (a)", true, ""},
-		{"CREATE INDEX idx ON t (a) USING HASH", true, ""},
-		{"CREATE INDEX idx ON t (a) COMMENT 'foo'", true, ""},
-		{"CREATE INDEX idx ON t (a) USING HASH COMMENT 'foo'", true, ""},
-		{"CREATE INDEX idx ON t (a) LOCK=NONE", true, ""},
-		{"CREATE INDEX idx USING BTREE ON t (a) USING HASH COMMENT 'foo'", true, ""},
-		{"CREATE INDEX idx USING BTREE ON t (a)", true, ""},
+		{"CREATE INDEX idx ON t (a)", true, "CREATE INDEX `idx` ON `t` (`a`)"},
+		{"CREATE INDEX idx ON t (a) USING HASH", true, "CREATE INDEX `idx` ON `t` (`a`) USING HASH"},
+		{"CREATE INDEX idx ON t (a) COMMENT 'foo'", true, "CREATE INDEX `idx` ON `t` (`a`) COMMENT 'foo'"},
+		{"CREATE INDEX idx ON t (a) USING HASH COMMENT 'foo'", true, "CREATE INDEX `idx` ON `t` (`a`) USING HASH COMMENT 'foo'"},
+		{"CREATE INDEX idx ON t (a) LOCK=NONE", true, "CREATE INDEX `idx` ON `t` (`a`)"},
+		{"CREATE INDEX idx USING BTREE ON t (a) USING HASH COMMENT 'foo'", true, "CREATE INDEX `idx` ON `t` (`a`) USING HASH COMMENT 'foo'"},
+		{"CREATE INDEX idx USING BTREE ON t (a)", true, "CREATE INDEX `idx` ON `t` (`a`) USING BTREE"},
```

Comment:
Can we also add a test case involving `CREATE UNIQUE INDEX`? I assume the parser can already parse it.

Reply:
done PTAL thanks